cmd/cli/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I think we should just use one single license file, it can be done in follow up PR